Objects/bytesobject.c

```diff
@@ -3744,12 +3744,49 @@ _PyBytes_Resize(PyObject **pv, Py_ssize_t newsize)
     return 0;
 }
 
+/* The following initializes `nullstring' and `characters' to avoid races
+   caused by lazily computing them.  Effectively, we're pessimizing any
+   programs that don't use empty or single character bytestrings, but it's a
+   one time cost and the upside is thread safety.
+ */
+int
+_PyBytes_Init(void)
+{
+    /* Inline PyObject_NewVar */
```

Comment:
I'm not quite sure of the convention here - this comment probably belongs further down in the function?